src/DssDirectDepositAaveDai.t.sol

```diff
@@ -109,18 +122,18 @@ contract DssDirectDepositAaveDaiTest is DSTest {
     function setUp() public {
         hevm = Hevm(address(bytes20(uint160(uint256(keccak256('hevm cheat code'))))));
 
-        chainlog = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+        chainlog = ChainlogLike(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
         vat = VatAbstract(0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B);
         end = EndAbstract(0xBB856d1742fD182a90239D7AE85706C2FE4e5922);
-        pool = LendingPoolLike(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9);
-        adai = DSTokenAbstract(0x028171bCA77440897B824Ca71D1c56caC55b68A3);
-        stkAave = DSTokenAbstract(0x4da27a545c0c5B758a6BA100e3a049001de870f5);
-        dai = DaiAbstract(0x6B175474E89094C44Da98b954EedeAC495271d0F);
-        daiJoin = DaiJoinAbstract(0x9759A6Ac90977b93B58547b4A71c78317f391A28);
-        interestStrategy = InterestRateStrategyLike(0xfffE32106A68aA3eD39CcCE673B646423EEaB62a);
-        rewardsClaimer = RewardsClaimerLike(0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5);
+        pool = LendingPoolAbstract(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9);
+        adai = TokenLike(0x028171bCA77440897B824Ca71D1c56caC55b68A3);
+        stkAave = TokenLike(0x4da27a545c0c5B758a6BA100e3a049001de870f5);
+        dai = TokenLike(0x6B175474E89094C44Da98b954EedeAC495271d0F);
```

Comment:
Head's up `DaiAbstract` and `DSTokenAbstract` have different interfaces. The cast will probably work here, but we might want to be explicit here in case tests get expanded on.

Reply:
yeah, was wondering the same, just followed the code pattern for now, as it uses `TokenLike` also for Dai.
I agree on splitting those interfaces for compatibility in case of future test expansions.

Reply:
done in bd15e60